x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/messages/Messages.java

```diff
@@ -154,7 +154,7 @@ public final class Messages {
     public static final String JOB_AUDIT_DATAFEED_NO_DATA = "Datafeed has been retrieving no data for a while";
     public static final String JOB_AUDIT_DATAFEED_MISSING_DATA =
         "Datafeed has missed {0} documents due to ingest latency, latest bucket with missing data is [{1}]."
-            + " Consider increasing query_delay";
+            + " Check if your ingest pipeline delays ingesting. If the pipeline works as expected, consider increasing query_delay.";
```

Comment:
I have no strong feelings about this. But IDK how common it is for folks to have ingest pipelines. 

Would a user be confused if they have no ingest pipelines?

If y'all think this is fine, I am cool with it. @droberts195

Reply:
I think there's been a misunderstanding at some point. "Ingest pipeline" here really means "end to end ingestion process". So that includes everything from:

- Originating application creating message
- Data collector reading the message
- Data collector processing
- Data collector sending to Elasticsearch
- Elasticsearch processing
- Elasticsearch indexing
- Elasticsearch making searchable

Using dictionary definitions of words that could be described as an "ingest pipeline", but we shouldn't use the term here because it has a very specific meaning in Elasticsearch.

So basically "ingest pipeline" should be replaced with something else throughout this PR. Maybe "end to end ingestion process", "end to end ingestion flow", "ingestion flow from source to index", or something better.

Reply:
Thanks for the input, I rephrased the text via [af50d7f](https://github.com/elastic/elasticsearch/pull/97753/commits/af50d7f38ef8906cdce92d1550103da065545b4b).